src/main/java/cn/nukkit/Player.java

```diff
@@ -2794,6 +2796,15 @@ public void onCompletion(Server server) {
 
                         this.dataPacket(entityEventPacket);
                         Server.broadcastPacket(this.getViewers().values(), entityEventPacket);
+                    } else if (entityEventPacket.event == EntityEventPacket.ENCHANT) {
+                        if (entityEventPacket.eid != this.id) {
+                            break;
+                        }
+
+                        Inventory inventory = this.getWindowById(ANVIL_WINDOW_ID);
+                        if (inventory instanceof AnvilInventory) {
+                            ((AnvilInventory) inventory).setCost(-entityEventPacket.data);
```

Comment:
I'm not entirely sure what this cost does but should we be taking the client's value or figuring it out our self?

Reply:
I think we should figure out how to calculate this cost